monkey/monkey_island/cc/services/agent_plugin_service/agent_plugin_service.py

```diff
@@ -71,22 +74,60 @@ def install_agent_plugin_archive(self, agent_plugin_archive: bytes):
                     operating_system=operating_system, agent_plugin=agent_plugin
                 )
 
-    def get_available_plugins(self, force_refresh: bool) -> AgentPluginRepositoryIndex:
+    def install_agent_plugin_from_repository(
```

Comment:
Suggestion: Since this is already `AgentPluginService`, we can save a few characters:
```suggestion
    def install_plugin_from_repository(
```

Reply:
We should also then fix `install{_agent,}_plugin_archive()`